Documentation/network/concepts/ipam/azure.rst

```diff
@@ -14,7 +14,10 @@ Azure IPAM
 
    While still maintained for now, Azure IPAM is considered legacy and is not
    compatible with AKS clusters created in `Bring your own CNI <https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli>`_
-   mode. Using BYOCNI is the preferred way to install Cilium on AKS.
+   mode. The recommended way to install cilium on AKS are 
+   `Bring your own CNI <https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli>`_ or
+   `Azure CNI Powered by Cilium <https://aka.ms/aks/cilium-dataplane>`_.
```

Comment:
```suggestion
   `Bring your own CNI <https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli>`__ or
   `Azure CNI Powered by Cilium <https://aka.ms/aks/cilium-dataplane>`__.
```

Can you add an extra underscore at the end of these links? That should fix the linting issues in the Documentation workflow.